pt-br/java-pt.html.markdown

```diff
@@ -7,6 +7,7 @@ contributors:
 translators:
     - ["Victor Kléber Santos L. Melo", "http://victormelo.com.br/blog"]
     - ["Renê Douglas N. de Morais", "mailto:rene.douglas.bsi@gmail.com"]
+    - ["Yuri Alessandro D. T. Martins", "https://github.com/YuriAlessandro"]
```

Comment:
LGTM, but we usually reserve the contributor tags to more significant contributions (like a second PR, a few paragraphs, ...)